deployments/optimism-goerli/usdc/deploy.ts

```diff
@@ -0,0 +1,135 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet, exp, wait } from '../../../src/deploy';
+
+const clone = {
+  op: '0x4200000000000000000000000000000000000042'
+};
+
+const secondsPerDay = 24 * 60 * 60;
+
+const GOERLI_TIMELOCK = '0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399';
```

Comment:
Curious if local timelock on optimism-goerli should be 'https://goerli-optimism.etherscan.io/address/0x4c8011724f2ece7b6ce4b13c90fe5036c9e1eb2d' instead of 0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399 ? 🤔 

Went to check 0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399, but it got empty address.

Reply:
So this is the Ethereum Goerli timelock, not the Optimism Goerli timelock 😃

We hardcode it here because it is provided as a constructor arg to the `BridgeReceiver`, which needs to know what L1 contract messages should be coming from.